webdriver-spec.html

```diff
@@ -1211,9 +1211,9 @@
  </tr>
 
  <tr>
-  <td><dfn>invalid element coordinates</dfn>
+  <td><dfn>invalid coordinates</dfn>
   <td>400
-  <td><code>invalid element coordinates</code>
+  <td><code>invalid coordinates</code>
```

Comment:
Not sure I understand why this change was made. Just a feeling that the word "element" was redundant? I'd prefer to avoid interop issues with existing clients and selenium implementations unless there's a good reason for it.